src/.env

```diff
@@ -14,3 +14,9 @@ CG_TOPIC_NAME_PROVISIONING=cloudengineering.confluentgateway.provisioning
 CG_TOPIC_NAME_MESSAGE_CONTRACT=cloudengineering.selfservice.messagecontract
 CG_TOPIC_NAME_SCHEMA=cloudengineering.confluentgateway.schema
 DEFAULT_KAFKA_BOOTSTRAP_SERVERS=localhost:9092
+KAFKA_CLUSTER_PROD_BROKER=localhost:9092

```

Comment:
This is the same secret ad the `DEFAULT_KAFKA_BOOTSTRAP_SERVERS`.
However, one is a reference to where our servers communicate internally (prod kafka cluster) the other to references are for how to access our clusters in general.
There is an overlap, as there must be, but the wording on the new variables are better.

We could consider just keeping the new ones and make `DEFAULT` an alias